libbpf-rs/tests/bin/src/taskiter.bpf.c

```diff
@@ -0,0 +1,31 @@
+#include "vmlinux.h"
+
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_core_read.h>
+
+struct index_pid_pair {
+  uint32_t i;
+  pid_t pid;
+};
+
+static uint32_t i = 0;
+
+SEC("iter/task")
+int dump_pid(struct bpf_iter__task *ctx)
+{
+  struct seq_file *seq = ctx->meta->seq;
+  struct task_struct *task = ctx->task;
+
+  if (!task)
+    return 0;
+
+  struct index_pid_pair p;
```

Comment:
please stick to C89 conventions: all variables are declared at the top of a function or lexical block